src/_sass/_search.scss

```diff
@@ -0,0 +1,54 @@
+@use "./colours";
+
+.search-form {
+  display: flex;
+
+  &__input {
+    background-color: colours.$ivory;
+    border: 0.063rem solid colours.$navy;
+    color: colours.$navy;
+    line-height: 1.35;
+    margin-right: -0.063rem;
+    padding: 0.5rem 0.938rem;
+    width: 100%;
+  }
+
+  &__submit {
+    background-color: colours.$yellow-light;
+    border: 0.063rem solid colours.$navy;
```

Comment:
Re: rounded corners, I think this is just the default `outline` style (it's the same on the main site). I've had a quick play around with alternatives, and I don't love any of them (including the default with rounded corners!), but here's a demo

**First:** no outline, change border colour to yellow - I think this might be too subtle
**Second:** yellow outline, retaining the blue border - maybe this is okay, though still more subtle than the default
**Third:** default - rounded corners

Two and three (default) have the issue of the outline extending beyond the height of the adjacent search button, and the first also looks like it does this, though it shouldn't (if I've recalled it correctly above).

Any thoughts/other suggestions?

https://user-images.githubusercontent.com/40244233/203781537-696fcb83-d897-484c-8e7b-daf1d95d79ad.mov

Reply:
Updated with what I think is a cleaner implementation: outlining the entire search form in the same light blue as underlines links (also used for anchor links)